pipelines/main/misc/fullsrcdist.yml

```
@@ -0,0 +1,33 @@
+steps:
+  - group: "Full Source Dist" # TODO: delete this line
+  # - group: "Check" # TODO: uncomment this line
+    steps:
+      - label: "fullsrcdist"
+        key: "fullsrcdist"
+        soft_fail: true # TODO: delete this line
+        plugins:
+          - JuliaCI/external-buildkite#v1:
+              version: "./.buildkite-external-version"
+              repo_url: "https://github.com/JuliaCI/julia-buildkite"
+          - JuliaCI/julia#v1:
+              # Drop default "registries" directory, so it is not persisted from execution to execution
+              persist_depot_dirs: packages,artifacts,compiled
+              version: '1.6'
+          - staticfloat/sandbox#v1:
+              rootfs_url: https://github.com/JuliaCI/rootfs-images/releases/download/v4.8/package_linux.x86_64.tar.gz
+              rootfs_treehash: "2a058481b567f0e91b9aa3ce4ad4f09e6419355a"
+              workspaces:
+                - "/cache/repos:/cache/repos"
+          - JuliaCI/julia#v1:
+              # Drop default "registries" directory, so it is not persisted from execution to execution
+              persist_depot_dirs: packages,artifacts,compiled
+              version: '1.6'
+        timeout_in_minutes: 10
+        commands: |
+          make full-source-dist USE_BINARYBUILDER=0
```

```suggestion
          make full-source-dist USE_BINARYBUILDER=0
          make light-source-dist USE_BINARYBUILDER=0
```
This would be convenient so that we can upload as release assets on GitHub.